src/lifecycles/lifecycle.helpers.js

```diff
@@ -6,7 +6,9 @@ export function validLifecycleFn(fn) {
   return fn && (typeof fn === "function" || isArrayOfFns(fn));
 
   function isArrayOfFns(arr) {
-    return Array.isArray(arr) && !find(arr, item => typeof item !== "function");
+    return (
+      Array.isArray(arr) && !find(arr, (item) => typeof item !== "function")
+    );
```

Comment:
this is one of the "bigger" changes I've seen in the prettier update. just an interesting note, no need to change anything.